tests/gocase/util/server.go

```diff
@@ -59,8 +60,13 @@ func (s *KvrocksServer) NewTCPClient() *tcpClient {
 }
 
 func (s *KvrocksServer) Close() {
-	require.NoError(s.t, s.cmd.Process.Kill())
-	require.EqualError(s.t, s.cmd.Wait(), "signal: killed")
+	require.NoError(s.t, s.cmd.Process.Signal(syscall.SIGTERM))
```

Comment:
I am wondering, if kvrocks crashed, whether it will report an error? cc @tisonkun

Reply:
I see the `s.cmd.Wait` below so it should be fine.